tests/Utils/Ethereum.js

```diff
@@ -59,7 +59,8 @@ function getContractDefaults() {
 }
 
 function keccak256(values) {
-  return ethers.utils.keccak256(values);
+  // alternatively use `return ethers.utils.keccak256(values)`
+  return web3.utils.soliditySha3(values);
```

Comment:
Isn't there also `web3.utils.keccak256`?

Reply:
Ah missed the comment, yes that seems better?

Reply:
good catch, I am not sure what is the right thing here. I can use https://web3js.readthedocs.io/en/v1.2.7/web3-utils.html#sha3 as an exact replacement. 

Also, ethers library has `solidityKeccak256`